packages/protocol/contracts/thirdparty/LibSecureMerkleTrie.sol

```diff
@@ -67,3 +67,27 @@ library LibSecureMerkleTrie {
         return abi.encodePacked(keccak256(_key));
     }
 }
+
```

Comment:
It will be great if we have a github action to enforce the license "header"

Reply:
yes, i'm going to open a separate issue for it. at the moment there is a compiler warning generated if it is missing, but no failure on CICD. we should check it is MIT compatible and that the SPDX header exists for solidity contracts.